cms/server/contest/templates/contest.html

```diff
@@ -174,35 +178,35 @@ <h3 id="countdown_box">
                                 {{ t_iter.name }}
                             </li>
                             <li{% if request.path == '/tasks/%s/description' % encode_for_url(t_iter.name) %} class="active"{% endif %}>
-                                <a href="{{ contest_url("tasks", t_iter.name, "description") }}">{{ _("Statement") }}</a>
+                                <a href="{{ contest_url("tasks", t_iter.name, "description") }}">{% trans %}Statement{% endtrans %}</a>
                             </li>
                             <li{% if request.path == '/tasks/%s/submissions' % encode_for_url(t_iter.name) %} class="active"{% endif %}>
-                                <a href="{{ contest_url("tasks", t_iter.name, "submissions") }}">{{ _("Submissions") }}</a>
+                                <a href="{{ contest_url("tasks", t_iter.name, "submissions") }}">{% trans %}Submissions{% endtrans %}</a>
                             </li>
         {% endfor %}
     {% endif %}
                             <li class="divider"></li>
                             <li{% if request.path == '/documentation' %} class="active"{% endif %}>
-                                <a href="{{ contest_url("documentation") }}">{{ _("Documentation") }}</a>
+                                <a href="{{ contest_url("documentation") }}">{% trans %}Documentation{% endtrans %}</a>
                             </li>
     {% if actual_phase == 0 or current_user.unrestricted %}{# FIXME maybe >= 0? #}
         {% if testing_enabled %}
                             <li{% if request.path == '/testing' %} class="active"{% endif %}>
-                                <a href="{{ contest_url("testing") }}">{{ _("Testing") }}</a>
+                                <a href="{{ contest_url("testing") }}">{% trans %}Testing{% endtrans %}</a>
                             </li>
         {% endif %}
         {% if printing_enabled %}
                             <li{% if request.path == '/printing' %} class="active"{% endif %}>
-                                <a href="{{ contest_url("printing") }}">{{ _("Printing") }}</a>
+                                <a href="{{ contest_url("printing") }}">{% trans %}Printing{% endtrans %}</a>
                             </li>
         {% endif %}
     {% endif %}
                         </ul>
                     </div>
                     <span class="license_notice">
-                    <a href="http://github.com/cms-dev/cms/" rel="author noreferrer" target="_blank">{{ _("Contest Management System") }}</a>
-                    {{ _("is released under the") }}
-                    <a href="http://www.gnu.org/licenses/agpl" rel="license noreferrer" target="_blank">{{ _("GNU Affero General Public License") }}</a>
+                    <a href="http://github.com/cms-dev/cms/" rel="author noreferrer" target="_blank">{% trans %}Contest Management System{% endtrans %}</a>
```

Comment:
Do we actually translate that?! It also probably needs to be translated in one piece instead of 3...

Reply:
Yes, and most locales even provide a translation. I don't know which are the other two places we localize it at, but if the messages are identical they appear as one in the .po files, which is OK.